modules/core/src/v2/keychains.ts

```diff
@@ -181,7 +181,7 @@ Keychains.prototype.add = function(params, callback) {
   .nodeify(callback);
 };
 
-Keychains.prototype.createBitGo = function(params, callback) {
+Keychains.prototype.createBitGo = function(params: any, callback: NodeCallback<any>): any {
```

Comment:
Doc comment please, also I think we can probably do better than `any` for the types here as well. Just adding any as the type is not really any better than leaving it alone, since it is an any type by default.

Reply:
its meant to be empty, but I can't set it to {} because on line 187 it sets ```params.source``` which is not allowed